content/rs/installing-upgrading/configuring/cluster-lba-setup.md

```diff
@@ -0,0 +1,100 @@
+
+When DNS cannot be used, clients can still connect to the endpoints with the IP addresses,
+but the benefits of load balancing and automatically updates IP address are missing.
+
+## Network architecture with load balancer
+
+You can compensate for the lack of DNS resolution with load balancers that can expose services and provide service discovery.
+A load balancer is configured in front of Redis Enterprise cluster, exposing several logical services:
+
+- Control plane services, such as the RS admin console to access cluster administration interface
+- Data plane services, such as a database endpoint to connect from client applications
+
+Depending on which Redis Enterprise services you want to access outside the cluster you may need to configure the load balancers seperately.
+One or more Virtual IPs (VIPs) are defined on the load balancer to expose Redis Enterprise services.
+The architecture is shown in the following diagram with 3 nodes Redis Enterprise cluster with one database (DB1) configured on port 12000:
+
+<to be added - image>
+
+## Setting up an RS cluster with load balancers
+
+### Prerequisites
+
+- [Install]({{< relref "/rs/installing-upgrading/_index.md" >}}) the latest version of RS on your clusters
+- Configure the cluster with the cluster name (FQDN) even though DNS is not in use.
+    Remember that the same cluster name is used to issue the licence keys.
+    We recommend that you use a “.local” suffix in the FQDN.
+
+### Configuring the load balancers
+
+- Make sure that the load balancer is performing TCP health checks on the cluster nodes.
+- Expose the services that you require through a Virtual IP, for example:
+    - Web Management Portal (8443)
+    - Rest API service (Secured - 9443; Non-secured - 8080)
+    - Database ports (In the range of 10000-19999)
+
+Other ports are shown in the list of [RS network ports]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}}).
+
+{{< note >}}
+Make sure the RS admin console only uses sticky, secured connections on port 8443:
+
+- Certain LBAs provide specific logic to close idle connections. Either disable this feature or make sure the applications connecting to Redis use reconnection logic.
+- Make sure the LBA is fast enough to resolve connections between two clusters or applications that are connected to Redis databases through LBA.
+- Choose the standard LBA which is commonly used in your environment so that you have easy access to in-house expertise for troubleshooting issues.
+{{< /note >}}
+
+### RS cluster configuration
+
+There are certain recommended settings within the cluster that guarantee a flawless connectivity experience for applications and admin users when they access the cluster through a Load Balancer.
+
+{{< note >}}
+- Run the `rladmin` commands directly on the cluster.
+- The `rladmin` commands update the settings on all nodes in the cluster.
+{{< /note >}}
+
+To optimize the cluster to run behind an LBA, run:
```

Comment:
Why would this be better off than dense placement and all-master-shards?
It might be a workaround for limited LBAs but I think the different options should at least be discussed here.

Reply:
Yes, we can elaborate on that point. The main idea is to provide a configuration that is "good enough" and provides easier integration behind a Load Balancer. We can add more options and at least insist on "handle_redirects enabled" setting.